src/commands/helpcenter.js

```diff
@@ -0,0 +1,51 @@
+const { SlashCommandBuilder } = require('@discordjs/builders');
+const { MessageEmbed } = require('discord.js');
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('helpcenter')
+    .setDescription('Provies information to access the helcenter!')
+    .addStringOption((option) =>
+      option
+        .setName('plaintext')
+        .setDescription('Provides helpcenter in plaintext format')
+        .setRequired(false)
+    ),
+
+  async execute(interaction) {
+    if (
+      interaction.options.getString('plaintext') == 'plaintext' ||
+      interaction.options.getString('plaintext') == 'true'
+    ) {
+      await interaction.reply({
+        content:
+          '**Codecademy Help Center:** https://help.codecademy.com/hc/en-us\n' +
+          '**Submit A Request:** https://help.codecademy.com/hc/en-us/requests/new\n' +
+          '**Bug Reporting:** To report a bug, click *Get Unstuck* in the learning environment, then click *Bugs*.\n' +
+          'All billing-related queries should be directed to the Submit A Request form linked above.',
+        fetchReply: true,
```

Comment:
We only need to fetch the reply if we're going to do something with it later (like update it).  Since we don't need that here, let's get rid of this line.